rest/dao/src/main/resources/io/syndesis/dao/deployment.json

```diff
@@ -755,31 +756,100 @@
+              "properties": {
+                  "query":{
+                    "kind":"path",
+                    "displayName":"Query",
+                    "group":"common",
+                    "required":true,
+                    "type":"string",
+                    "javaType":"java.lang.String",
+                    "deprecated":false,
+                    "secret":false,
+                   "description":"SQL Statement to be executed. Can contain input parameters prefixed by ':#' i.e. ':#MYPARAMNAME'"
+                 }
+               }
+            }]
+          }
+        },
+        {
+          "id": "sql-start-connector",
+          "name": "Periodic SQL Invocation",
+          "description": "Invoke SQL to periodically get data out",
+          "tags": [ "dynamic" ],
+          "pattern": "From",
+          "actionType": "connector",
+          "descriptor": {
+            "camelConnectorGAV": "io.syndesis:sql-start-connector:1.2-SNAPSHOT",
+            "camelConnectorPrefix": "sql-start-connector",
+            "inputDataShape": {
+               "kind" : " none"
+            },
+            "outputDataShape": {
+               "kind" : "json-schema"
+            },
+            "propertyDefinitionSteps": [{
+              "name": "SQL Statement",
+              "description": "Enter a SQL Statement starting with SELECT",
+              "properties": {
+                  "query":{
+                    "kind":"path",
+                    "displayName":"Query",
+                    "group":"common",
+                    "required":true,
+                    "type":"string",
+                    "javaType":"java.lang.String",
+                    "deprecated":false,
+                    "secret":false,
+                   "description":"SQL Select Statement to be executed"
+                 },
+                  "schedulerPeriod":{
+                    "kind":"parameter",
+                    "displayName":"Period",
+                    "group":"consumer",
+                    "required":false,
+                    "type":"integer",
+                    "javaType":"long",
+                    "deprecated":false,
+                    "secret":false,
+                    "defaultValue":1000,
+                    "description":"Delay in milli seconds between scheduling (executing)"
+                  }
+               }
+            }]
+          }
+        },
+        {
+          "id": "sql-stored-connector",
+          "name": "Invoke Stored Procedure",
+          "description": "Invoke a Stored Procedure to get Data in and/or out",
           "tags": [ "dynamic" ],
           "pattern": "To",
           "actionType": "connector",
           "descriptor": {
-            "camelConnectorGAV": "io.syndesis:sql-stored-connector:@syndesis-connectors.version@",
+            "camelConnectorGAV": "io.syndesis:sql-stored-connector:1.2-SNAPSHOT",
```

Comment:
How to deal with depending on a SNAPSHOTS? How does this work in the mono repo? Hardcoding for now..